rust/schema/annotations.py

```diff
@@ -0,0 +1,1741 @@
+from misc.codegen.lib.schemadefs import *
+from .ast import *
+
+
+@annotate(Module)
+@rust.doc_test_signature(None)
+class _:
+    """
+    A module declaration. For example:
+    ```rust
+    mod foo;
+    ```
+    ```rust
+    mod bar {
+        pub fn baz() {}
+    }
+    ```
+    """
+
+
+@annotate(Expr)
+class _:
+    """
+    The base class for expressions.
+    """
+    pass
```

Comment:
nit: `pass` is not required, and currently the file is not very consistent on it :)